proto-public/pbauth/v1alpha1/traffic_permissions.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
```

Comment:
Are you planning on creating `TrafficPermissions` in a follow-up PR?

Reply:
I'm guess this is `NamespacedTrafficPermission` but my tendency would be to simplify as `TrafficPermission`. Otherwise it might be confusing for CE users.

Reply:
renamed it to just TrafficPermission, but it would be nice to figure out a naming solution or alias for ENT so it is clear that it doesn't encompass PartitionTrafficPermission